### ISSUE TITLE
what about the performance

### ISSUE THREAD

**Issue by argman** [BowieHsu/tensorflow_ocr#1]
nice to have pixel-link implemented, what about the performance on icdar ?

**Comment by BowieHsu** [comment]
@argman for now just 60% f-score train from scratch mode based on ICDAR2015 datasets, and learning rate is 0.0001, I think the bad performance may caused by label compute procedure and loss function calculation.
